### PR TITLE
Database Set-up

### PR DESCRIPTION
**Pull request by rpire**
## Changes made:
- Created `patients`, `medical_histories`, `treatments`, `invoices`, and `invoice_items` tables.
- Added relationships between tables using `foreign keys`.
- Created a `join table` called `diagnosis` to handle M:M relationships between the `medical_histories` and `treatments` tables.

### CODE REVIEW

File: schema-based-on-diagram.sql [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  treatment_id INT,
+
+  CONSTRAINT fk_medical_histories
+  FOREIGN KEY (medical_id)
+  REFERENCES medical_histories(id),
+
+  CONSTRAINT fk_treatments
+  FOREIGN KEY (treatment_id)
+  REFERENCES treatments(id)
+);
```
Comment (AshakaE) [inline schema-based-on-diagram.sql]: all FOREIGN KEYS have indexes created

- [x] Kindly update SQL commands

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @rpire  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

###  Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] 

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DammyShittu** [comment]
Hello @AshakaE, 👋 

Thank you for reviewing our pull request.

We will work on the changes and request another review as soon as possible.

Cheers 🎉

File: schema-based-on-diagram.sql (hunk 1 above)
Comment (rpire) [reply schema-based-on-diagram.sql]: Required changes completed.

**Review by AshakaE — APPROVED** [review]
### Hi @rpire & @DammyShittu  🙋‍♀️, 

# STATUS APPROVED ✔
Thanks for making the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by rpire** [comment]
Hello, @AshakaE 👋

Thank you again for reviewing our work. We will proceed to merge these changes into the `dev` branch. 🚀

Best regards!

**Pull request merged**